package/MDAnalysis/topology/ITPParser.py

```diff
@@ -268,7 +268,7 @@ def find_path(self, path):
             path = path.name
         except AttributeError:
             pass
-
+        path = os.path.abspath(path)
```

Comment:
More of a style thing, but can this be just done on line 268 rather than assigning path twice in a row?